notebook/static/notebook/js/actions.js

```diff
@@ -545,13 +552,13 @@ define(function(require){
         },
     };
 
-    // private stuff that prepend `.ipython` to actions names
+    // private stuff that prepend `jupyter-notebook:` to actions names
     // and uniformize/fill in missing pieces in of an action.
     var _prepare_handler = function(registry, subkey, source){
-        registry['ipython.'+subkey] = {};
-        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
-        registry['ipython.'+subkey].help_index = source[subkey].help_index;
-        registry['ipython.'+subkey].icon = source[subkey].icon;
+        registry['jupyter-notebook:'+subkey] = {};
+        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
+        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
+        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
```

Comment:
In the long term, we will augment commands with a one-line description, which can be used for presentation in the UI. We don't want to choose overly verbose command names now, just for the sake of UI presentation.